llvm/lib/Transforms/Instrumentation/HWAddressSanitizer.cpp

```diff
@@ -337,13 +337,17 @@ class HWAddressSanitizer {
                                  unsigned AccessSizeIndex,
                                  Instruction *InsertBefore, DomTreeUpdater &DTU,
                                  LoopInfo *LI);
-  bool ignoreMemIntrinsic(MemIntrinsic *MI);
+  bool ignoreMemIntrinsic(OptimizationRemarkEmitter &ORE, MemIntrinsic *MI);
   void instrumentMemIntrinsic(MemIntrinsic *MI);
   bool instrumentMemAccess(InterestingMemoryOperand &O, DomTreeUpdater &DTU,
                            LoopInfo *LI);
   bool ignoreAccess(Instruction *Inst, Value *Ptr);
+  bool ignoreAccessWithRemark(OptimizationRemarkEmitter &ORE, Instruction *Inst,
```

Comment:
I would prefer without redundant `WithRemark` just 'ignoreAccess(OptimizationRemarkEmitter &ORE,'

Reply:
OK. Swapped the other one to be `WithoutRemark`. I can also do overloading, but that makes it easier to accidentally use the wrong one. But I don't care either way.